clinica/pipelines/t1_linear/t1_linear_pipeline.py

```diff
@@ -0,0 +1,223 @@
+       RegistrationSynQuick from ANTs.
+    3) Crop the background (in order to save computational power).
+
+    Returns:
+        A clinica pipeline object containing the T1 Linear pipeline.
+    """
+
+    def check_custom_dependencies(self):
+        """Check dependencies that can not be listed in the `info.json` file.
+        """
+        pass
+
+    def get_input_fields(self):
+        """Specify the list of possible inputs of this pipeline.
+
+        Returns:
+            A list of (string) input fields name.
+        """
+
+        return ['t1w']
+
+    def get_output_fields(self):
+        """Specify the list of possible outputs of this pipeline.
+
+        Returns:
+            A list of (string) output fields name.
+        """
+
+        return ['image_id']
+
+    def build_input_node(self):
+        """Build and connect an input node to the pipeline.
+        """
+        from os import pardir
+        from os.path import dirname, join, abspath, split, exists
+        import sys
+        import nipype.interfaces.utility as nutil
+        import nipype.pipeline.engine as npe
+        from clinica.utils.inputs import check_bids_folder
+        from clinica.utils.exceptions import ClinicaBIDSError, ClinicaException
+        from clinica.utils.inputs import clinica_file_reader
+        from clinica.utils.input_files import T1W_NII
+        from clinica.utils.inputs import fetch_file
+        from clinica.utils.stream import cprint
+
+        root = dirname(abspath(join(abspath(__file__), pardir, pardir)))
+        path_to_mask = join(root, 'resources', 'masks')
+        self.ref_template = join(
+                path_to_mask, 'mni_icbm152_t1_tal_nlin_sym_09c.nii')
+        self.ref_crop = join(path_to_mask, 'ref_cropped_template.nii.gz')
+        url1 = "https://aramislab.paris.inria.fr/files/data/img_t1_linear/ref_cropped_template.nii.gz"
+        url2 = "https://aramislab.paris.inria.fr/files/data/img_t1_linear/mni_icbm152_t1_tal_nlin_sym_09c.nii"
+
+        if not(exists(self.ref_template)):
+            try:
+                fetch_file(url2, self.ref_template)
+            except IOError as err:
+                cprint('Unable to download required template (mni_icbm152) for processing:', err)
+
+        if not(exists(self.ref_crop)):
+            try:
+                fetch_file(url1, self.ref_crop)
+            except IOError as err:
+                cprint('Unable to download required template (ref_crop) for processing:', err)
+
+        # Inputs from anat/ folder
+        # ========================
+        # T1w file:
+        try:
+            t1w_files = clinica_file_reader(
+                    self.subjects,
+                    self.sessions,
+                    self.bids_directory,
+                    T1W_NII)
+        except ClinicaException as e:
+            err = 'Clinica faced error(s) while trying to read files in your BIDS directory.\n' + str(e)
+            raise ClinicaBIDSError(err)
+
```

Comment:
Currenly, nothing is displayed when running the pipeline:
```text
$ clinica run t1-linear BIDS_T1_LINEAR/ CAPS_T1_LINEAR -np 2 -wd WD
[14:08:10] The t1-linear pipeline has completed. You can now delete the working directory (/home/alexandre.routier/Temp/LightDataset/WD/t1-linear).
```
The minimum information I currently demand is to display the image(s) to process and add estimation of time computation. You can simply add these lines:

```python
        if len(self.subjects):
            print_images_to_process(self.subjects, self.sessions)
            cprint('The pipeline will last approximately 42 minutes per image.') # Replace by adequate computational time.
```

These lines were added to the Jinja template (PR #63) but after you generated the skeleton.